### DISCUSSION TITLE
Site taiga-ui.dev

### DISCUSSION THREAD

**Discussion by ghost** [taiga-family/taiga-ui#182] [Ideas]
Hello

1. Your site is developing intensively and I would like that in the left menu, if there is new information, then there was a label - new or update.
Otherwise, it is difficult to understand - what's new?
2. I would like a development plan, for example, for a quarter - what to expect in development Taiga-ui?
3. When changing a dark theme to a light one, not everything is updated on the page and some of the information is not visible.

Thanks :-)

**Comment by MarsiBarsi** [comment]
Hey Vladislav 👋
Thank you for your interest in our library!

1. We have [CHANGELOG.md](https://github.com/TinkoffCreditSystems/taiga-ui/blob/main/CHANGELOG.md) file and [Changlog page](https://taiga-ui.dev/changelog) at our site with all relevant changes
2. All right, the first step of our roadmap is creating a roadmap :) We'll do it soon. Now I can say only that the next big feature is addon-table
3. Thanks, we'll take a look
